src/basic/cgroup-util.c

```diff
@@ -1295,23 +1298,28 @@ static const char *skip_user_manager(const char *p) {
         p += strspn(p, "/");
 
         n = strcspn(p, "/");
-        if (n < STRLEN("user@x.service"))
+        if (n < STRLEN("user@x.service")) /* || n < STRLEN("capsule@x.service"))*/
+                return NULL;
+        unit_name = strndupa(p, n);
```

Comment:
Not a matter, but please insert an empty line after `return NULL;`
Also, use `const char *unit_name`, if it will not be modified.